src/pyenphase/models/meters.py

```diff
@@ -0,0 +1,44 @@
+"""Model for the Envoy's CT Meters."""
+from __future__ import annotations
+
+from enum import StrEnum
+
+
+class EnvoyPhaseMode(StrEnum):
+    SPLIT = "split"
+    THREE = "three"
+    SINGLE = "single"
+
+
+class EnvoyMeterType(StrEnum):
+    PRODUCTION = "production"
+    NETCONSUMPTION = "net-consumption"
+    TOTALCONSUMPTION = "total-consumption"
+
+
+class EnvoyMeterState(StrEnum):
+    ENABLED = "enabled"
+    DISABLED = "disabled"
+
+
+class EnvoyMeteringStatus(StrEnum):
+    NORMAL = "normal"
+    NOTMETERING = "not-metering"
+    CHECKWIRING = "check-wiring"
+
+
+class EnvoyMeterStatusFlags(StrEnum):
+    PODUCTIONIMBALANCE = "production-imbalance"
```

Comment:
```suggestion
    PRODUCTION_IMBALANCE = "production-imbalance"
```

Please separate words with `_`